### DISCUSSION TITLE
Anyone play Half-Life?

### DISCUSSION THREAD

**Discussion by 737simpilot** [CIDRAM/CIDRAM#344] [General]
Anyone play Half-Life? Just saw a blocked IP out of China that belongs to this ASN. Company is obscure as all hell.

 

![img](https://i.postimg.cc/SN0S0YpN/erdfgbheh.jpg)

**Comment by Maikuolan** [comment]
Lol. Used to play, a long, long time ago. Not for many, many years now though.
